.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Summary
+
+(Summarize the problem solved by this PR, and how to verify it manually)
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New scenario (non-breaking change which adds functionality)
+- [ ] This change requires a documentation update
+
+### Checklist:
+- [ ] Methods and classes used in PR scenarios are meaningful according to the purpose of this test
+- [ ] External tickets are updated according to these changes
+- [ ] Commits are well encapsulated and follow [team agreetments](https://gitlab.cee.redhat.com/quarkus-qe/signpost/-/blob/main/pull-requests.md)
```

Comment:
I don't really like this for multitude of reasons:

* this is an opensource project, but the doc is closed behind VPN
* we're exposing internal infra 
* we have contributors not in the team